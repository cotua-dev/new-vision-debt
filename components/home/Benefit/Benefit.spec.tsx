import { render, RenderResult } from '@testing-library/react';
import { Benefit } from './index';

describe('Benefit Component', (): void => {
    let renderResult: RenderResult;

    beforeAll((): RenderResult => renderResult = render(<Benefit/>));

    test('renders', (): void => {
        const { container } = renderResult;
        expect(container.firstChild).toMatchSnapshot();
    });

    afterAll((): void => renderResult.unmount());
});
