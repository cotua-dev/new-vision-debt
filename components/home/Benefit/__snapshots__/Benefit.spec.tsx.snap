// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Benefit Component renders 1`] = `
<section
  class="benefit"
>
  <h2
    class="benefit-title"
  >
    Here's How You Would Benefit
  </h2>
  <div
    class="benefit-table-wrapper"
  >
    <table
      class="benefit-table"
    >
      <caption
        class="benefit-table-caption"
      >
        <strong
          class="benefit-table-caption-text"
        >
          If you had a total debt of...
        </strong>
        <br />
        <strong
          class="benefit-table-caption-text"
        >
          $25,000
        </strong>
      </caption>
      <thead>
        <tr>
          <th />
          <th>
            <strong
              class="benefit-table-head-text"
            >
              Monthly Payment
            </strong>
          </th>
          <th>
            <strong
              class="benefit-table-head-text"
            >
              Total Repayment
            </strong>
          </th>
          <th>
            <strong
              class="benefit-table-head-text"
            >
              Debt Free In
            </strong>
          </th>
          <th>
            <strong
              class="benefit-table-head-text"
            >
              Interest Rate
            </strong>
          </th>
          <th>
            <strong
              class="benefit-table-head-text"
            >
              1-On-1 Expert Help
            </strong>
          </th>
        </tr>
      </thead>
      <tbody>
        <tr>
          <td>
            <picture>
              <source />
              <img
                alt="Debt Advisors Of America Regular Logo"
                class="w-36 mx-auto"
                loading="lazy"
              />
            </picture>
          </td>
          <td>
            <strong
              class="debt-advisors-data"
            >
              $408
            </strong>
          </td>
          <td>
            <strong
              class="debt-advisors-data"
            >
              $18,750
            </strong>
          </td>
          <td>
            <strong
              class="debt-advisors-data"
            >
              46 months
            </strong>
          </td>
          <td>
            <strong
              class="debt-advisors-data"
            >
              0%
            </strong>
          </td>
          <td>
            <svg
              aria-hidden="true"
              class="svg-inline--fa fa-check w-6 h-6 text-green-700 mx-auto"
              data-icon="check"
              data-prefix="fas"
              focusable="false"
              role="img"
              viewBox="0 0 448 512"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M438.6 105.4C451.1 117.9 451.1 138.1 438.6 150.6L182.6 406.6C170.1 419.1 149.9 419.1 137.4 406.6L9.372 278.6C-3.124 266.1-3.124 245.9 9.372 233.4C21.87 220.9 42.13 220.9 54.63 233.4L159.1 338.7L393.4 105.4C405.9 92.88 426.1 92.88 438.6 105.4H438.6z"
                fill="currentColor"
              />
            </svg>
          </td>
        </tr>
        <tr>
          <td>
            <strong
              class="debt-advisors-row-label"
            >
              Consolidation Loan
            </strong>
          </td>
          <td>
            <p>
              $676
            </p>
          </td>
          <td>
            <p>
              $40,580
            </p>
          </td>
          <td>
            <p>
              60 months
            </p>
          </td>
          <td>
            <p>
              21%
            </p>
          </td>
          <td>
            <svg
              aria-hidden="true"
              class="svg-inline--fa fa-xmark w-6 h-6 text-red-700 mx-auto"
              data-icon="xmark"
              data-prefix="fas"
              focusable="false"
              role="img"
              viewBox="0 0 320 512"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M310.6 361.4c12.5 12.5 12.5 32.75 0 45.25C304.4 412.9 296.2 416 288 416s-16.38-3.125-22.62-9.375L160 301.3L54.63 406.6C48.38 412.9 40.19 416 32 416S15.63 412.9 9.375 406.6c-12.5-12.5-12.5-32.75 0-45.25l105.4-105.4L9.375 150.6c-12.5-12.5-12.5-32.75 0-45.25s32.75-12.5 45.25 0L160 210.8l105.4-105.4c12.5-12.5 32.75-12.5 45.25 0s12.5 32.75 0 45.25l-105.4 105.4L310.6 361.4z"
                fill="currentColor"
              />
            </svg>
          </td>
        </tr>
        <tr>
          <td>
            <strong
              class="debt-advisors-row-label"
            >
              Making Only Minimum Payments
            </strong>
          </td>
          <td>
            <p>
              $729
            </p>
          </td>
          <td>
            <p>
              $72,262
            </p>
          </td>
          <td>
            <p>
              230 months
            </p>
          </td>
          <td>
            <p>
              23%
            </p>
          </td>
          <td>
            <svg
              aria-hidden="true"
              class="svg-inline--fa fa-xmark w-6 h-6 text-red-700 mx-auto"
              data-icon="xmark"
              data-prefix="fas"
              focusable="false"
              role="img"
              viewBox="0 0 320 512"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M310.6 361.4c12.5 12.5 12.5 32.75 0 45.25C304.4 412.9 296.2 416 288 416s-16.38-3.125-22.62-9.375L160 301.3L54.63 406.6C48.38 412.9 40.19 416 32 416S15.63 412.9 9.375 406.6c-12.5-12.5-12.5-32.75 0-45.25l105.4-105.4L9.375 150.6c-12.5-12.5-12.5-32.75 0-45.25s32.75-12.5 45.25 0L160 210.8l105.4-105.4c12.5-12.5 32.75-12.5 45.25 0s12.5 32.75 0 45.25l-105.4 105.4L310.6 361.4z"
                fill="currentColor"
              />
            </svg>
          </td>
        </tr>
      </tbody>
    </table>
  </div>
  <p
    class="benefit-content"
  >
    * The minimum payment example is based on a credit card for individuals with fair credit, having an interest rate of 23% (rounded up to the nearest percentage point) and assumes a payment of 3% of the balance. Please note that, assuming the principal balance does not increase due to additional charges, fees, and interest, the required minimum monthly payment will decrease over time as additional minimum monthly payments are made and reduce the total balance.
  </p>
  <strong
    class="benefit-content"
  >
    Our goals are to help you resolve your enrolled debts as quickly as possible and to get you into a plan with one of our licensed partners to negotiate settlements for less than what you owe
  </strong>
  <a
    class="start-reducing-payments-link"
    href="/stepper"
  >
    Start Reducing Your Payments
  </a>
  <p
    class="benefit-content"
  >
    We've helped our previous referrals achieve major savings
  </p>
</section>
`;
